### PR TITLE
Prevents automatic logging of ClickHouse Authorization information

### PR DESCRIPTION
**Pull request by xurui-c**
We don't want consumers to log ClickHouse username and password, otherwise ClickHouse servers' security could be compromised.

### CODE REVIEW

File: rust_snuba/src/consumer.rs [hunk 1]
```diff
@@ -80,7 +83,26 @@ pub fn consumer_impl(
...
+            storage.clickhouse_cluster.http_port,
+            storage.clickhouse_cluster.database,
+        );
+        if debug_mode {
+            tracing::info!(
+                "ClickHouse username: {}, ClickHouse password: {}",
+                storage.clickhouse_cluster.user,
+                storage.clickhouse_cluster.password
+            );
+        }
```
Comment (volokluev) [inline rust_snuba/src/consumer.rs:103]: I would remove this alltogether

File: rust_snuba/src/consumer.rs [hunk 2]
```diff
@@ -80,7 +83,26 @@ pub fn consumer_impl(
...
+        //tracing::info!(storage.message_processor, "Message Processor")
```
Comment (volokluev) [inline rust_snuba/src/consumer.rs:104]: don't leave comments in code :)

**Review by volokluev — COMMENTED** [review]
personally I don't think the debug mode is necessary, leaves room for a security vulnerability

File: rust_snuba/src/consumer.rs (hunk 2 above)
Comment (evanh) [inline rust_snuba/src/consumer.rs:104]: Is this supposed to be commented?

Comment (xurui-c) [reply rust_snuba/src/consumer.rs]: I forgot to remove this line all together

**Review by volokluev — APPROVED** [review]

**Review by anonrig — APPROVED** [review]

**Pull request merged**